### ISSUE TITLE
Black loading screen 

### ISSUE THREAD

**Issue by SShiifty** [Cuffuffles/CClient#5]
Everytime i launch the game i get stuck on a black loading screen and i have already unistalled the game and reinstalled it

**Comment by The-Solid-lad** [comment]
+1
have the same problem, attempted to fix it the same way to no avail

**Comment by Cuffuffles** [comment]
Yeah unfortunately the latest game update broke it. Trying to work out the best way around it since even if I can fix it and release an update, it can't be downloaded if it won't load.

**Comment by The-Solid-lad** [comment]
You'll probably have to implement auto-updates on startup, to avoid stuff like this in the future and just tell everyone to DL the new version of the client.
Many ppl know about the Cclient because of Mosswi, so if u asked him to make a youtube vid about this, I'm sure those people would notice and do so.

**Comment by Cuffuffles** [comment]
Yeah, I purposely wanted updating to be optional in case an update was broken or there was an update loop, but it looks like that kinda backfired lmao

The updater should (maybe) still work, since while you can't see it, you can still click the version number to open the changelog in the bottom right. I'll do my best haha
